.github/workflows/cd.yml

```diff
@@ -9,9 +9,19 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: DerYeger/pnpm-setup-action@master
```

Comment:
Is is just disliking someone's workflow?
Or enabling the cache? We should create our own then.

Reply:
It's because the one that is used is not allowing to select which version of PNPM to use and is 2 years without any updates. The one I used is an official Github action of PNPM

Reply:
It's because the one that is used is not allowing to select which version of PNPM to use and is 2 years without any updates. The one I used is an official Github action of PNPM

Reply:
That sounds fair. I wonder if you'd be interested in creating an updated action. Or you prefer using the official version and copying yaml configs to all jotaijs/* repos?

Reply:
I would prefer to use the official version. And about copying. I can do that, but later. I'm well overdue with this task and want to focus on work at the company where I'm working for a bit :)